configs/components/facter.json

```diff
@@ -1 +1 @@
-{"url":"git@github.com:puppetlabs/facter.git","ref":"81a6e127335edd2d045a606a56c6c63d5347179d"}
+{"url":"git@github.com:puppetlabs/facter.git","ref":"4.5.2"}
```

Comment:
I don’t think this is needed either as 81a6 is what we tagged at 4.5.2, https://github.com/puppetlabs/facter/releases/tag/4.5.2. My recollection is the pipeline tags the components and later automatically replaces the sha with the corresponding tag